pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -390,6 +421,20 @@ func (p *DNSProxy) ServeDNS(w dns.ResponseWriter, request *dns.Msg) {
 	}
 }
 
+func lookupDestIPID(destIPStr string) (SecID ipcache.Identity, exists bool, err error) {
+	maxPrefixLen := 32
```

Comment:
```suggestion
	maxPrefixLen := 8 * net.IPv4len
```

Reply:
oops, yes. I saw these and simply forgot :/